client/entrypoints/express-checkout/index.js

```diff
@@ -140,6 +143,7 @@ jQuery( function ( $ ) {
 				currency: options.currency,
 				paymentMethodCreation: 'manual',
 				appearance: getExpressCheckoutButtonAppearance(),
+				paymentMethodTypes: getPaymentMethodTypes(),
```

Comment:
❓ I'm not sure if this is an issue or not but this `getPaymentMethodTypes()` will return all enabled payment method types including those that aren't related to express payment buttons. 
eg 

```
Array(4)
0: "card"
1: "eps"
2: "sepa_debit"
3: "link"
```

Reply:
Good catch, James! I have introduced a new function to return only what we need here:
```
export const getExpressPaymentMethodTypes = ( paymentMethodType = null ) =>
	getPaymentMethodTypes( paymentMethodType ).filter( ( type ) =>
		[
			'link',
			'google_pay',
			'apple_pay',
			'paypal',
			'amazon_pay',
			'klarna',
		].includes( type )
	);
```